externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/events/SessionEvent.scala

```diff
@@ -43,6 +43,7 @@ case class SessionEvent(
     username: String,
     ip: String,
     serverIp: String,
+    conf: Map[String, String],
```

Comment:
I think this is connection initializing config map, how about the configs from `set` command

Reply:
> I think this is connection initializing config map, how about the configs from `set` command

The set statement will be displayed in the statement list, we don't need to override it here, it might be better to show the initial configurations.

Reply:
Yes, can't agree more. Comment on purpose 😁